py3dtiles/gltf_material.py

```diff
@@ -0,0 +1,71 @@
+class GlTFMaterial():
+    def __init__(self, metallicFactor=0, roughnessFactor=0, rgb=[1, 1, 1], alpha=1):
+        self.metallicFactor = metallicFactor
+        self.roughnessFactor = roughnessFactor
+        self.rgba = rgb
+        if len(rgb) < 4:
+            self.alpha = alpha
+
+    @property
+    def metallicFactor(self):
+        return self._metallicFactor
+
+    @metallicFactor.setter
+    def metallicFactor(self, value):
+        if value > 1:
+            value = 1
+        if value < 0:
+            value = 0
+        self._metallicFactor = value
+
+    @property
+    def roughnessFactor(self):
+        return self._roughnessFactor
+
+    @roughnessFactor.setter
+    def roughnessFactor(self, value):
+        if value > 1:
```

Comment:
[DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) with metallicFactor() code.
If you wish to normalize define and use something [like NormalizeData()](https://stackoverflow.com/questions/18380419/normalization-to-bring-in-the-range-of-0-1) ?